src/example/counter/index.tsx

```diff
@@ -1,10 +1,10 @@
 import React from "react";
 import { useDispatch, useSelector } from "react-redux";
-import { RootState } from "../../store/reducer";
 import {
-  increamentCouter,
   decreamentCouter,
+  increamentCouter,
```

Comment:
```suggestion
  decrementCounter,
  incrementCounter,
```
please fix this typo